src/components/Menu/MenuItem/AvatarMenuItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { forwardRef, useCallback } from "react";
+import MenuItem from "./MenuItem";
+import Avatar from "../../Avatar/Avatar";
+import PropTypes from "prop-types";
+
+// MenuItem with Avatar instead of icon
+const AvatarMenuItem = forwardRef((props, ref) => {
```

Comment:
please deconstruct with `{ avatarProps, ...rest }`

Reply:
Could you please check [this commit](https://github.com/mondaycom/monday-ui-react-core/pull/706/commits/91825cd8a7f3951ea323c863064142c73daad9e8)? Does it look better? 
Code completion for MenuItem props is working fine, but for Avatar props it's not, how do I fix that?

Reply:
IMO i rather have al the props in the argument of the function and not to deconstruct it with the component